frontend/src/pages/Player/Toolbar/DevToolsWindowV2/utils.ts

```diff
@@ -118,37 +125,32 @@ export const LogLevelVariants = {
 
 export enum RequestType {
 	All = 'All',
-	Link = 'Link',
-	Script = 'Script',
-	Other = 'Other',
-	XHR = 'XHR',
-	CSS = 'CSS',
-	iFrame = 'iFrame',
-	Fetch = 'Fetch',
-	Img = 'Img',
-}
-
-export enum Tab {
-	Errors = 'Errors',
-	Console = 'Console',
-	Network = 'Network',
-	Events = 'Events',
+	Link = 'link',
+	Script = 'script',
+	Other = 'other',
+	XHR = 'xmlhttprequest',
+	CSS = 'css', 
+	iFrame = 'iframe', // didn't find a request to verify that 'iframe' is what is actually received
+	Fetch = 'fetch',
+	Img = 'img',
 }
 
```

Comment:
looks like lint fails on this file - you can run `yarn format:all` in the repo to lint everything.

![image](https://user-images.githubusercontent.com/1351531/222922425-92645777-92bf-4ab2-b3f1-79ca68b1af99.png)